### PR TITLE
Cat image gallery

### PR DESCRIPTION
**Pull request by gwynethsc**
Creates an separate linked page with a brief description of my cats, accompanied by a small photo gallery.

### CODE REVIEW

File: portfolio/src/main/webapp/cats.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+        <h1>meow.</h1>
+        <img class="title-image" src="images/cats/siblings.jpg">
+        <p>Meet Ender (the striped and white one) and Valentine (the floofy grey one). Yes, Ender and Valentine as in Ender's Game by Orson Scott Card - my dad named them. (My family also previously had two cats named Calvin and Hobbes.) Ender actually passed away last year, but he was a very playful and affectionate cat; he'd often climb into my lap or behind my back when I sat on the sofa and rub his face against mine. Valentine's a bit more reserved, but she still loves to be brushed and isn't shy about asking for it! My family is full of cat people; I'm fairly certain that the humans meow more than the cats!</p>
+        <div class="image-gallery-row">
+            <div class="image-gallery-column">
+                <a href="images/cats/enderpeek.jpg"><img src="images/cats/enderpeek.jpg" title="I am in this photo on the far left"</a>
```
Comment (sperzion) [inline portfolio/src/main/webapp/cats.html:16]: Did you include these images in the commit?

File: portfolio/src/main/webapp/cats.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8">
+    <title>Cats</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+        <h1>meow.</h1>
+        <img class="title-image" src="images/cats/siblings.jpg">
+        <p>Meet Ender (the striped and white one) and Valentine (the floofy grey one). Yes, Ender and Valentine as in Ender's Game by Orson Scott Card - my dad named them. (My family also previously had two cats named Calvin and Hobbes.) Ender actually passed away last year, but he was a very playful and affectionate cat; he'd often climb into my lap or behind my back when I sat on the sofa and rub his face against mine. Valentine's a bit more reserved, but she still loves to be brushed and isn't shy about asking for it! My family is full of cat people; I'm fairly certain that the humans meow more than the cats!</p>
```
Comment (sperzion) [inline portfolio/src/main/webapp/cats.html:13]: Consider formatting so you don't have to scroll to read all the content. I know this is done automatically when it's rendered, but it's harder for you and reviewers to maintain!

Comment (gwynethsc) [reply portfolio/src/main/webapp/cats.html]: Done!

File: portfolio/src/main/webapp/cats.html (hunk 1 above)
Comment (gwynethsc) [reply portfolio/src/main/webapp/cats.html:21]: Apparently I somehow actually deleted the image folder altogether . . . fixed!

Comment (sperzion) [reply portfolio/src/main/webapp/cats.html:21]: Also, according to https://github.com/gwynethsc/step/blob/master/walkthroughs/week-2-web-development/portfolio-walkthrough.md#uploading-images it says that you should use "/images" note the leading forward slash, as that's supposed to work in both development and production.

Comment (gwynethsc) [reply portfolio/src/main/webapp/cats.html]: Thanks for the reminder! I believe I've changed them all now.

**Pull request merged**